examples/platforms/debian/default

```diff
@@ -31,7 +31,7 @@
 NAT64=0
 DNS64=0
 DHCPV6_PD=0
-DHCPV6_PD_REF=1
+DHCPV6_PD_REF=0
```

Comment:
Can you confirm that it's feasible to make a change in ot-reference-release repo to enable `DHCPV6_PD_REF`?

Reply:
Yes, I already made the change https://github.com/openthread/ot-reference-release/blob/57722d0e3812da9a895e2faeea86cd4668f3528c/script/otbr-setup.bash#L170